### PR TITLE
Add New Featured Artist: cygnus news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎵

### CODE REVIEW

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2023/2023-05-13-new-featured-artist-cygnus.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Evolution%20of%20War%2Fcygnus%20-%20STRIKER%2B.mp3">
+</audio>
+
+### cygnus - The Evolution of War
+
+Resident 4K Featured Artist mapper [FAMoss](https://osu.ppy.sh/users/7707789) is here to deliver [this map](https://osu.ppy.sh/beatmapsets/1987180) to all of osu!'s piano smashers.
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Evolution%20of%20War%2Fcygnus%20-%20The%20Evolution%20of%20War.mp3">
```
Comment (Walavouchey) [inline news/2023/2023-05-13-new-featured-artist-cygnus.md:38]: what's up with the backslashes being encoded as `%2F`? https://assets.ppy.sh/artists/359/Evolution%20of%20War/cygnus%20-%20The%20Evolution%20of%20War.mp3 is equivalent and is how it has been done before

from a cursory look over fa posts during last year this seems to have been going on for a while, but it seems to be a bit random for some reason, with both ways appearing in the same article such as [this one](https://github.com/ppy/osu-wiki/blob/master/news/2022/2022-07-27-new-featured-artist-pegboard-nerds.md?plain=1#L46-L54)

i may do a pr to clean these up at some point, but it's fairly inconsequential

Comment (Ephemeralis) [reply news/2023/2023-05-13-new-featured-artist-cygnus.md]: sometimes i overshoot when using vscode's urlencode plugin, that's all it is

fairly inconsequential either way i think, though `%2F` should probably always be `/` just for readability's sake

[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2023/2023-05-13-new-featured-artist-cygnus.md]: have done this

File: news/2023/2023-05-13-new-featured-artist-cygnus.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        <source src="https://assets.ppy.sh/artists/359/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### cygnus - STRIKER+
+
+Play through [this soon-to-be-Ranked map](https://osu.ppy.sh/beatmapsets/1966073) with an 8-star top difficulty and guest difficulties by the *no caps crew*: [rrtyui](https://osu.ppy.sh/users/352328), [mithew](https://osu.ppy.sh/users/2571893), [semaphore](https://osu.ppy.sh/users/6313643), and [fanzhen0019](https://osu.ppy.sh/users/418699)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/The%20Evolution%20of%20War/cygnus%20-%20STRIKER%2B.mp3">
```
Comment (Walavouchey) [inline news/2023/2023-05-13-new-featured-artist-cygnus.md:30]: this one is broken now here and on the website (seems to be still located at the old url https://assets.ppy.sh/artists/359/Evolution%20of%20War%2Fcygnus%20-%20STRIKER%2B.mp3)

File: news/2023/2023-05-13-new-featured-artist-cygnus.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+</audio>
+
+### cygnus - The Evolution of War
+
+Resident 4K Featured Artist mapper [FAMoss](https://osu.ppy.sh/users/7707789) is here to deliver [this map](https://osu.ppy.sh/beatmapsets/1987180) to all of osu!'s piano smashers.
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/The%20Evolution%20of%20War/cygnus%20-%20The%20Evolution%20of%20War.mp3">
```
Comment (Walavouchey) [inline news/2023/2023-05-13-new-featured-artist-cygnus.md:38]: fine here but broken on the website (with a typo https://assets.ppy.sh/artists/359/The%20Evolution%20of%20War/cygnus%20-%20The%20The%20Evolution%20of%20War.mp3)

image

**Review by peppy — APPROVED** [review]

File: news/2023/2023-05-13-new-featured-artist-cygnus.md (hunk 3 above)
Comment (Ephemeralis) [reply news/2023/2023-05-13-new-featured-artist-cygnus.md:38]: fixed on web side

File: news/2023/2023-05-13-new-featured-artist-cygnus.md (hunk 2 above)
Comment (Ephemeralis) [reply news/2023/2023-05-13-new-featured-artist-cygnus.md:30]: i think this is a caching thing on your end, will purge before merge just in case

Comment (Ephemeralis) [reply news/2023/2023-05-13-new-featured-artist-cygnus.md:30]: (checked, all fixed)

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**